#Try getting an input inside the try catch block

try:
    age=int(input('Enter your age: '))
except:
    print ('You have entered an invalid value.')
